dags/bqetl_amo_stats.py

```diff
@@ -84,19 +96,35 @@
 
     amo_dev__amo_stats_dau__v2.set_upstream(amo_prod__amo_stats_dau__v2)
 
-    wait_for_main_summary_clients_daily = ExternalTaskSensor(
-        task_id="wait_for_main_summary_clients_daily",
-        external_dag_id="main_summary",
-        external_task_id="clients_daily",
+    wait_for_telemetry_derived__clients_daily__v6 = ExternalTaskSensor(
+        task_id="wait_for_telemetry_derived__clients_daily__v6",
+        external_dag_id="bqetl_clients_daily",
+        external_task_id="telemetry_derived__clients_daily__v6",
```

Comment:
We haven't merged https://github.com/mozilla/telemetry-airflow/pull/1047, so this DAG doesn't exist in telemetry-airflow, yet. (I guess we should have waited with merging it in bigquery-etl too, since now all generated DAGs will assume `bqetl_clients_daily` actually exists...).

```suggestion
        external_dag_id="main_summary",
        external_task_id="clients_daily",
```